EXAMPLES.md

```diff
@@ -277,9 +277,16 @@ additionalArguments:
 - "--experimental.plugins.bouncer.version=v1.1.9"
 ```
 
-# Use Traefik Let's Encrypt Integration with CloudFlare
+# Use Traefik native Let's Encrypt Integration, without cert-manager
+
+This example is with CloudFlare, see here for other providers.
```

Comment:
`see here` - I think we need to add a link here? 😄

Reply:
Yes, I added it.